src/Sema.zig

```diff
@@ -18780,22 +18780,21 @@ fn addToInferredErrorSet(sema: *Sema, uncasted_operand: Air.Inst.Ref) !void {
         .adhoc_inferred_error_set_type => {
             const ies = sema.fn_ret_ty_ies.?;
             assert(ies.func == .none);
-            try addToInferredErrorSetPtr(mod, ies, sema.typeOf(uncasted_operand));
+            try addToInferredErrorSetPtr(sema.arena, mod, ies, sema.typeOf(uncasted_operand));
         },
         else => if (ip.isInferredErrorSetType(err_set_ty)) {
             const ies = sema.fn_ret_ty_ies.?;
             assert(ies.func == sema.func_index);
-            try addToInferredErrorSetPtr(mod, ies, sema.typeOf(uncasted_operand));
+            try addToInferredErrorSetPtr(sema.arena, mod, ies, sema.typeOf(uncasted_operand));
         },
     }
 }
 
-fn addToInferredErrorSetPtr(mod: *Module, ies: *InferredErrorSet, op_ty: Type) !void {
-    const gpa = mod.gpa;
+fn addToInferredErrorSetPtr(arena: Allocator, mod: *Module, ies: *InferredErrorSet, op_ty: Type) !void {
```

Comment:
nit: since you're having to take the Sema arena and this function is in Sema.zig, I'd just replace the first two params here with `sema: *Sema`

Reply:
Done!